app/views/course/admin/admin/index.html.slim

```diff
@@ -15,6 +15,7 @@
     = f.input :start_at
     = f.input :end_at
     = f.input :gamified, hint: t('.gamified_label')
+    = f.input :braincert_api_key, placeholder: t('.braincert_api_key_hint')
```

Comment:
This field should only be disabled if `Lecture` is enabled, and I think we should move this to the Lecture component settings.